packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.gts

```diff
@@ -29,16 +33,52 @@ export default class SchedulePaymentFormActionCard extends Component<Signature>
   }
   @tracked selectedPaymentType: string | undefined;
   @action onSelectPaymentType(paymentTypeId: string) {
+    if (paymentTypeId === 'one-time' && !this.paymentDate) {
+      this.paymentDate = new Date();
+    }
+    if (paymentTypeId === 'monthly') {
+      if (!this.monthlyUntil) {
+        let now = new Date();
+        this.monthlyUntil = new Date(now.getFullYear() + 1, now.getMonth(), now.getDate());
+      }
+      if (!this.paymentDayOfMonth) {
+        this.paymentDayOfMonth = 1;
+      }
+    }
```

Comment:
These seem like things to have assertions on, do you agree? Though maybe it’ll be difficult with ✨ time zones ✨